docs/supported-mutators.md

```diff
@@ -320,3 +321,61 @@ Strings and literals identified to a regex are mutated in the following way:
 | `--a`    | `++a`   |
 
 [🔝 Back to Top](#)
+
+## Math Methods
+
+> Currently only implemented for Stryker.NET. Future implementations for other languages can be implemented differently.
+
+| Original                   | Mutated                    |
+| -------------------------- | -------------------------- |
+| `Acos()`                   | `Acosh()`                  |
```

Comment:
If you're checking if this method comes from the `Math` class, shouldn't these be `Math.Acos()` -> `Math.Acosh()` ?

Reply:
Done